renku/core/management/dataset/dataset_add.py

```diff
@@ -191,14 +201,19 @@ def _process_urls(
                             "information and updates."
                         )
                     u = urllib.parse.urlparse(url)
-                    new_files = _add_from_local(
+                    new_files, warnings = _add_from_local(
                         client, dataset=dataset, path=u.path, external=external, destination=destination
                     )
+                    tracked_external_warnings.extend(warnings)
                 else:  # Remote URL
                     new_files = _add_from_url(client=client, url=url, destination=destination, extract=extract)
 
             files.extend(new_files)
 
+    if tracked_external_warnings:
+        message = "\n  ".join(tracked_external_warnings)
+        communication.warn(f"Warning: The following files cannot be added as external:\n  {message}")
```

Comment:
```suggestion
        message = "\n\t".join(tracked_external_warnings)
        communication.warn(f"Warning: The following files cannot be added as external:\n\t{message}")
```
I think in most places we use the Tab character for horizontal alignment, should probably also use it here for consistency.